README.md

```diff
@@ -119,7 +119,7 @@ Settings for the `ServiceWorker` cache. Use `null` or `false` to disable `Servic
 * `output`: `string`. Relative (from the _webpack_'s config `output.path`) output path for emitted script.  
 _Default:_ `'sw.js'`
 
-* `entry`: `string`. Relative or absolute path to file which will be used as `ServiceWorker` entry. Useful to implement additional function or handle other SW events.  
+* `entry`: `string`. Relative or absolute path to the file which will be used as the `ServiceWorker` entry/bootstrapping. Useful to implement additional features or handlers for Service Worker events.  
```

Comment:
Probably something like "handlers for Service Worker events such as `push`, `sync`, etc"